frontend/src/bundles/common/components/header/header.tsx

```diff
@@ -63,7 +63,8 @@ const Header: React.FC<Properties> = ({
                 {token ? (
                     <div className={classNames(styles.headerBody, styles.tabs)}>
                         {(pathname === AppRoute.DASHBOARD ||
-                            pathname === AppRoute.BUDGETS) && (
+                            pathname === AppRoute.BUDGETS ||
+                            pathname.match(/^\/budgets\/\d+$/)) && (
```

Comment:
I recomend to set this regex to variable

Reply:
resolved in [https://github.com/BinaryStudioAcademy/bsa-winter-2022-2023-smartspend/pull/286/commits/b20760f84841bd84bb032c5ba118188e58a721e5](url)